whitepapers/0010_ibc_generic_messaging.md

```diff
@@ -0,0 +1,92 @@
+# IBC Bridging
+
+## Objective
+
+Since Wormchain is a cosmos-sdk based chain that is IBC-enabled, we can leverage IBC generic messaging to reduce the operational burden required to run a guardian node. Wormhole guardians should, therefore, be capable of scaling to support all IBC-enabled chains at minimal cost while only running a full node for Wormchain.
+
+## Background
+
+[IBC](https://ibcprotocol.org/) is the canonical method of generic message passing within the cosmos ecosystem. IBC is part of the cosmos-sdk and Cosmos chains can enable it to connect with other cosmos chains.
+
+[Wormchain](https://github.com/wormhole-foundation/wormhole/tree/main/wormchain) is a cosmos-sdk based blockchain that has been purpose-built to support Wormhole. It allows the guardians to store global state on all the blockchains which wormhole connects to, and enables a suite of product and security features.
+
+## Goals
+
+- Remove the requirement for guardians to run full nodes for IBC-enabled chains. Guardians should be able to support any IBC-enabled chain by only running a full wormchain node.
+- Define a custom IBC specification for passing wormhole messages between IBC chains and wormchain.
```

Comment:
Technically the channel is bi-directional, but in practice it is not because the `wormchain-ibc-receiver` contract does not send any IBC packets.

AFAIK directionality restrictions are not built into the IBC protocol itself. They are an application-level detail based on which side is able to emit IBC packets.